docs/debug.md

```diff
@@ -1,3 +1,60 @@
+  * Otherwise, Grafana is able to fetch data, and there is an issue with the dashboards specifically. Continue to the next step.
+* Check if the `Label` and `Value` fields are correctly populated in the top left corner. If they contain data, well, it should be working... Otherwise continue to the next step.
+* These fields are populated using the `acm_managed_cluster_labels` metric and the `observability-managed-cluster-label-allowlist` configmap. This configmap is read by the `rbac-query-proxy`. Check if you see error logs in the `rbac-query-proxy` pod that points to an issue with those. You can restart it to make sure they have latest configmap data.
+* Check if the `acm_managed_cluster_labels` is collected by ACM from the grafana explorer. If not, make sure it is collected by the metrics collector on the hub.
+  * Chech the list of metrics in the `metrics-collector` pod args.
+  * Make sure that the `acm_managed_cluster_labels` metric is in the `observability-metrics-allowlist` configmap in the `metrics_list.yaml` file.
+* Check if the `acm_managed_cluster_labels` is collected on the hub by the in-cluster prometheus. You can use the console for this.
+  * If it is not collected, move to the __Prometheus scraping issues__ section. Bear in mind that the `acm_managed_cluster_labels` metric is managed by the MCE team. It is scraped from the `clusterlifecycle-state-metrics` pod in the `multicluster-engine` namespace.
+
+## Prometheus scraping issues
+
+You expect some metrics in Prometheus but they are not there, and prometheus is up and working. You can do some of the following checks from the UI. But we focus here on debugging using shells. You can run the commands from the console, getting a terminal from the target pod, or from your local machine using `oc exec -n openshift-monitoring prometheus-k8s-0 -c prometheus -- `. Here are some steps to debug the issue:
+
+* Check if prometheus can reach the target by fetching the `metrics` path of the target from the prometheus pod in the `openshift-monitoring` namespace:
+```bash
+curl http://<TARGET_SERVICE>:<TARGET_PORT>/metrics
+```
+  Make sure that you find the metrics you are looking for.
+  * If it fails check the network policies.
+* Check if the `serviceMonitor` for the target exists and is correctly configured. Make sure that relabelling rules are not impacting your metrics.
+* Check the currently loaded configuration of the prometheus pod to see if you find the scraping job named after it, and if it is correctly configured:
+```bash
+curl http://localhost:9090/api/v1/status/config | sed 's/\\n/\n/g' | grep "job_name: serviceMonitor/NAMESPACE/SERVICE_MONITOR_NAME" -A100 
+# or from your local machine, pipe in less and search for the job name
+curl -s http://localhost:9090/api/v1/status/config | jq -r '.data.yaml' | yq '.scrape_configs' | less
+```
+* Check the state of the target:
```

Comment:
I found out that this is also visible in the openshift console under Observe->targets (see the screenshot in https://issues.redhat.com/browse/ACM-14296), we could add a small reference to this here, as I found that was actually a pretty easy way to check if the target is OK. One can also use that page to check if the target is in the scrape config.

Reply:
Nice, adding that